test/e2e/e2e_suite_test.go

```diff
@@ -53,9 +53,8 @@ func createTestCatalog(ctx context.Context, name string, imageRef string) (*cata
 			Source: catalogd.CatalogSource{
 				Type: catalogd.SourceTypeImage,
 				Image: &catalogd.ImageSource{
-					Ref:                   imageRef,
-					InsecureSkipTLSVerify: true,
```

Comment:
Need to deal with lack of `InsecureSkipTLSVerify` in E2Es somehow now.

Reply:
@joelanford's recent changes should've handled some of this... He added another CA to be trusted.

Reply:
It did, but on operator-controller side. This is catalogd. https://github.com/operator-framework/catalogd/pull/399 will help, but we still need to modify catalogd deployment on operator-catalog side to be able to use insecure TLS in E2E.